### ISSUE TITLE
module 'socket' has no attribute 'AF_UNIX'.

### ISSUE THREAD

**Issue by mohhamedss** [dbcli/mycli#554]
Hi guys!
Whenever I try to execute mycli from PS, I keep getting this response:
> module 'socket' has no attribute 'AF_UNIX'

Any suggestions?

P.S. I've uninstalled and reinstalled MYCLI and PYTHON.

**Comment by amjith** [comment]
I'm guessing PS is power shell. 

I'll check where we use the AF_UNIX socket. I don't have a Windows machine to debug this right now but I'll check out code to see if we have UNIX specific logic. 

Thanks for reporting.

**Comment by amjith** [comment]
A fix for this just landed on master (Thanks @tsroten). 

You can try it out by installing directly from master using the following command: 

```
pip install -U https://github.com/dbcli/mycli/archive/master.zip
```

Let us know if that works for you.

**Comment by mohhamedss** [comment]
That solved the problem!
Thank you so much for your quick response and the beautiful mycli!
;)

**Comment by tsroten** [comment]
@mohhamedss This bug fix was just released in mycli version 1.16.0. Thanks for reporting it!
